.github/workflows/main.yml

```diff
@@ -42,22 +42,32 @@ jobs:
     continue-on-error: true
     strategy:
       matrix:
-        protocol:
-          - kathmandunet
-          - limanet
+        include:
+          - testnet: kathmandunet
+            testnet_uppercase: KATHMANDUNET
+          - testnet: limanet
+            testnet_uppercase: LIMANET
     steps:
     - uses: actions/checkout@v3
     - uses: actions/setup-node@v3
       with:
         node-version: 16
+    # TODO: remove after debugging
```

Comment:
Should this be deleted?

Reply:
ah thanks for catching it! removed the debug statements